### ISSUE TITLE
Docker A2 GPU support

### ISSUE THREAD

**Issue by zidsi** [triton-inference-server/server#4287]
**Is your feature request related to a problem? Please describe.**
Docker (latest?) Release 20.03 (build 11042949) does not support A2. 

**Describe the solution you'd like**
I'd like A2 GPU to be supported in docker release ;)

**Additional context**
Docker started with textbook 
```
docker run --gpus=1 --rm --shm-size=1g --ulimit memlock=-1 --ulimit stack=67108864 -p8000:8000 -p8001:8001 -p8002:8002 -v/home/server/docs/examples/model_repository:/models nvcr.io/nvidia/tritonserver:20.03-py3 tritonserver --model-repository=/models
```
**WARNING: Detected NVIDIA NVIDIA A2 GPU, which is not yet supported in this version of the container** 
and (it might be unrelated): _/opt/tensorrtserver/nvidia_entrypoint.sh: line 92: exec: tritonserver: not found_

**Comment by dyastremsky** [comment]
Would you be able to try running 22.03? That would be the latest release.

20.03 is actually two years old (from March 2020), and your A2 GPU is newer.

**Comment by zidsi** [comment]
TLDR; It works with 22.03, thank you. 

How embarrassing and yet wonderful. Did pull 22.03 and than run "typo" 20.03. Again and again.
